packages/edit-site/src/components/style-book/index.js

```diff
@@ -321,7 +334,86 @@ function StyleBook( {
 	);
 }
 
-const StyleBookBody = ( {
+/**
+ * Style Book Preview component renders the stylebook without the Editor dependency.
+ *
+ * @param {Object}   props            Component props.
+ * @param {string}   props.path       Path to the selected block.
+ * @param {Object}   props.userConfig User configuration.
+ * @param {Function} props.isSelected Function to check if a block is selected.
+ * @param {Function} props.onSelect   Function to select a block.
+ * @return {Object} Style Book Preview component.
+ */
+export const StyleBookPreview = ( {
+	path = '',
+	userConfig = {},
+	isSelected,
+	onSelect,
+} ) => {
+	const siteEditorSettings = useSelect(
+		( select ) => select( siteEditorStore ).getSettings(),
+		[]
+	);
+	// Update block editor settings because useMultipleOriginColorsAndGradients fetch colours from there.
+	dispatch( blockEditorStore ).updateSettings( siteEditorSettings );
+
+	const [ resizeObserver, sizes ] = useResizeObserver();
+	const { colors, gradients } = useMultipleOriginColorsAndGradients();
+	// Exclude the default colors and gradients.
+	const themeColors = colors?.filter( ( color ) => color.slug === 'theme' );
+	const themeGradients = gradients?.filter(
+		( gradient ) => gradient.slug === 'theme'
+	);
+
+	const examples = getExamples( {
+		colors: themeColors,
+		gradients: themeGradients,
+		duotones: [], // Classic themes don't support duotone palettes.
```

Comment:
The classic theme can define duotone presets via theme.json, right?

Reply:
Hmm that's a good point, I might need to change the logic a bit.